src/test/unit/comparison.test.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import { getEmptyProfile } from '../../profile-logic/data-structures';
+import { mergeProfiles } from '../../profile-logic/comparison';
+
+describe('mergeProfiles function', function() {
+  it('should set interval of merged profile to minimum of all intervals', function() {
+    const profile1 = getEmptyProfile();
+    const profile2 = getEmptyProfile();
+    profile1.meta.interval = 10;
+    profile2.meta.interval = 20;
+
+    const mergedProfile = mergeProfiles([profile1, profile2], []);
```

Comment:
I think we need to pass valid profile states as the second argument. You can get valid states using the function `stateFromLocation` and passing a URL to it, eg `https://fakeurl.com/public/fakehash1/?thread=0&v=3`. `thread=0` makes it very explicit that we select the first thread of the profile (that is the only one we defined). I think you can use the same state twice, but you can as well define 2 different objects, I don't mind.

Reply:
@julienw, when I pass the given URL including https, the test shows the error `unexpected data source` and when I try to pass `/public/fakehash1/?thread=0&v=3` as the URL the test shows the error `No thread has been selected in profile ${i}`. 
Could you help me with this! 
Thanks.

Reply:
Ah right, `stateFromLocation` is called with an object that "look like" a Location object. For example:
```js
  const urlState = stateFromLocation({
    '/public/fakehash1/',
    search: '?thread=0&v=3',
    hash: '',
  });
```
or
```js
  const urlState = stateFromLocation(
    new URL('https://fakeurl.com/public/fakehash1/?thread=0&v=3')
  );
```

Reply:
Thanks!